pkg/ebpftracer/c/headers/common/network.h

```diff
@@ -9,70 +9,79 @@
 
 #include <common/common.h>
 
-// clang-format off
-
 // TYPES
 
 typedef union iphdrs_t {
     struct iphdr iphdr;
     struct ipv6hdr ipv6hdr;
 } iphdrs;
 
-typedef union  {
-    u32 v4addr;
-    unsigned __int128 v6addr;
-}  __attribute__((packed)) addr_t;
+typedef union {
+    __u8 raw[16];
+    __u32 v4addr;
+    __be32 u6_addr32[4];
+} __attribute__((packed)) addr_t;
 
 typedef struct {
     addr_t saddr;
     addr_t daddr;
-    u16 sport;
-    u16 dport;
-    u16 family;
+    __u16 sport;
```

Comment:
Why change these types to underscode?

Reply:
Just for consistencies sake. In the end both are equivalent.